### PR TITLE
SHO-32-fix: Paragraph

### PR DESCRIPTION
**Pull request by kevinochoag**
## Describe your changes

This PR adds a new component: paragraph

## Task URL (Asana, Jira, etc...)

[SHO-32](https://cultoperrocafe.nifty.pm/l/W3_NAvD4M7m9)

## What was done in this pull request

- [x] New component: `<Paragraph />`
- [x] Storybook with all variants

## Demo / Screenshot / Video

sho-32-fix

### CODE REVIEW

File: app/components/atoms/paragraph/Paragraph.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import styled, {css} from 'styled-components';
+
+export interface ParagraphVariant {
+  height: 'small' | 'large';
+  weight: 'normal' | 'bold'
+}
+
+const heights = {
+  small: css`
```
Comment (jmsalcido) [inline app/components/atoms/paragraph/Paragraph.styles.tsx]: there is no `normal` height?

**Review by jmsalcido — COMMENTED** [review]
do we **really** need a component for a paragraph?, I am okay with the code but not 100% sure we require a component for a paragraph, but anyway, looks good!

Lets wait to see what @tonymtz says.

File: app/components/atoms/paragraph/Paragraph.styles.tsx (hunk 1 above)
Comment (tonymtz) [reply app/components/atoms/paragraph/Paragraph.styles.tsx]: normal is a valid value for weight, not for height :)

**Comment by tonymtz** [comment]
I see why you added the props `height` and `width`, but the naming is not the best.
My main concern is in the way to consume this component as it's written right now:

**Scenario 1:**

```
<Paragraph text="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut ac tempus lacus. Proin mauris justo, porttitor ut odio a, mollis volutpat quam. Morbi dapibus arcu nec eros consectetur maximus. Duis pharetra odio rhoncus metus fringilla ultricies. Sed rhoncus dui quis turpis accumsan, nec volutpat ante consectetur. Suspendisse porta felis vitae dui blandit, quis dapibus arcu euismod. Nullam eget orci ut tellus blandit viverra. Nullam sodales a risus eu ultrices. Integer nec ligula tempor, vehicula urna ac, laoreet dui. Phasellus consequat sit amet nisi ut finibus." />
```

vs

```
<Paragraph>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut ac tempus lacus. Proin mauris justo, porttitor ut odio a, mollis volutpat quam. Morbi dapibus arcu nec eros consectetur maximus. Duis pharetra odio rhoncus metus fringilla ultricies. Sed rhoncus dui quis turpis accumsan, nec volutpat ante consectetur. Suspendisse porta felis vitae dui blandit, quis dapibus arcu euismod. Nullam eget orci ut tellus blandit viverra. Nullam sodales a risus eu ultrices. Integer nec ligula tempor, vehicula urna ac, laoreet dui. Phasellus consequat sit amet nisi ut finibus.
</Paragraph>
```

Passing long texts as children is easier to write and format. Why is this different from https://github.com/tepachelabs/perro-cafe-storefront/pull/4#discussion_r1145273594? We don't really care about the value of children in this context.

---

In the same train of thought:

**Scenario 2:**

```
<Paragraph text="Lorem ipsum dolor sit amet, consectetur adipiscing elit." weight="bold" height="large">
```

vs

```
<Paragraph bold lineHeight="1.5">
  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
</Paragraph>
```

---

The changes make sense, but I feel like sticking to something closer to the actual HTML tag would be more beneficial. Less friction when adopting our design system.

**Review by tonymtz — APPROVED** [review]

**Pull request merged**